models/models.js

```diff
@@ -10,7 +10,8 @@ var OrderSchema = new Schema({
 var IngredientSchema = new Schema({
     _creator : { type: Schema.ObjectId, ref: 'Order' },
```

Comment:
Not sure what you are using _creator for, you don't need to know who created an ingredient